quickstart/text.xml

```diff
@@ -397,20 +381,23 @@ DEPEND="${RDEPEND}
 	nls? ( >=sys-devel/gettext-0.12.1 )"
 
 src_prepare() {
-	epatch "${FILESDIR}"/${PN}-namespace.diff \
+	eapply "${FILESDIR}"/${PN}-namespace.diff \
 		"${FILESDIR}"/${PN}-procmime.diff
 }
 
 src_configure() {
-	econf \
-		$(use_enable nls) \
-		$(use_enable ssl) \
-		$(use_enable crypt gpgme) \
-		$(use_enable pda jpilot) \
-		$(use_enable ldap) \
-		$(use_enable ipv6) \
-		$(use_enable imlib) \
+	local myconf=(
+		$(use_enable nls)
+		$(use_enable ssl)
+		$(use_enable crypt gpgme)
+		$(use_enable pda jpilot)
+		$(use_enable ldap)
+		$(use_enable ipv6)
+		$(use_enable imlib)
 		$(use_enable xface compface)
+	)
+
+	econf "${myconf[@]}"
```

Comment:
That unnecessarily introduces a variable, and I think it is a matter of personal preference.
Also the existing example is easier to read (IMHO), so please leave it as it is.

Reply:
There is no harm in using a local variable. Relying on backslashes is harmful, especially when someone accidentally forgets a backslash in middle of it. And yes, I've seen such mistakes in practice.

Reply:
Also, case of passing flags directly to econf is presented in another example. I think it's beneficial to present more than one possible solution.

Reply:
I disagree. This is just your personal preference, not something that needs to be recommended in general. That there can be unrelated mistakes is not an argument. The variable could be misspelt as well, or the quotes be omitted or whatnot.

Reply:
Then just skip that one commit.